apps/meteor/tests/end-to-end/api/rooms.ts

```diff
@@ -1176,6 +1176,75 @@ describe('[Rooms]', () => {
 				})
 				.end(done);
 		});
+
+		it('should only delete file attachments when filesOnly is set to true', async () => {
+			const siteUrl = await getSettingValueById('Site_Url');
+			const message1Response = await sendSimpleMessage({ roomId: publicChannel._id });
+			const mediaResponse = await request
+				.post(api(`rooms.media/${publicChannel._id}`))
+				.set(credentials)
+				.attach('file', imgURL)
+				.expect('Content-Type', 'application/json')
+				.expect(200);
+
+			const message2Response = await request
```

Comment:
Let's separate this in a specific case, e.g: 'should not delete quote attachments when filesOnly is set to true'